atable/src/components/ATable.vue

```diff
@@ -14,16 +14,17 @@
 					:tableid="tableData.id">
 					<ACell
 						v-for="(col, colIndex) in tableData.columns"
-						:key="colIndex"
+						:key="`${colIndex}:${rowIndex}`"
 						:tableid="tableData.id"
 						:col="col"
 						spellcheck="false"
 						:rowIndex="rowIndex"
 						:colIndex="colIndex + (tableData.zeroColumn ? 0 : -1)"
 						:component="col.cellComponent"
 						:style="{
-							textAlign: col?.align?.toLowerCase() || 'center',
+							textAlign: col?.align || 'center',
 							minWidth: col?.width || '40ch',
+							width: tableData.config.fullWidth ? 'auto' : null,
```

Comment:
@Alchez `min-width`, then `width: auto` works perfectly. Your code for freezing the width of the first column if perfect. Try resizing the window in the full width story.

Reply:
Oh, I think I misunderstood how `min-width` and `width` work. That's a good fix!